### PR TITLE
Updated Peregrine base to allow to support peregrinecms-com-peregrine-sling-auth-header  module

### PR DESCRIPTION
**Pull request by GastonGonzalez**

### CODE REVIEW

File: admin-base/core/src/main/java/com/peregrine/admin/servlets/CreateTenantServlet.java [hunk 1]
```diff
@@ -201,7 +201,7 @@ protected Response handleRequest(Request request) throws IOException {
                 tenantGroup.addMember(tenantUser);
             } else {
                 // We also need to add the current non-admin user to the group so that new site is visible for them
-                Authorizable authorizable = userManager.getAuthorizable(userName);
+                Authorizable authorizable = userManager.getAuthorizable(request.getRequest().getUserPrincipal());
```
Comment (schaefa) [inline admin-base/core/src/main/java/com/peregrine/admin/servlets/CreateTenantServlet.java:202]: Can you remove the userName variable?

**Comment by reusr1** [comment]
@GastonGonzalez looking at the PR it seems it would mess up the ci/cd pipeline and the docker release part. Can you please make sure your changes here would be working correctly once the issues/93 branch gets merged into the develop branch? thank you

File: admin-base/core/src/main/java/com/peregrine/admin/servlets/CreateTenantServlet.java (hunk 1 above)
Comment (GastonGonzalez) [reply admin-base/core/src/main/java/com/peregrine/admin/servlets/CreateTenantServlet.java:202]: Yes. Will do.

**Comment by GastonGonzalez** [comment]
Per @reusr1 , merging PR.

**Pull request merged**